src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -5,9 +5,35 @@
  * Do not remove no-field constructor
  */
 public class Truck extends Machine {
+    private int cargoCapacity;
+    private int numberOfAxles;
+
+    public Truck(int cargoCapacity, int numberOfAxles) {
+        this.cargoCapacity = cargoCapacity;
+        this.numberOfAxles = numberOfAxles;
+    }
+
     public Truck() {
     }
 
+    public int getCargoCapacity() {
+        return cargoCapacity;
+    }
+
+    public Truck setCargoCapacity(int cargoCapacity) {
+        this.cargoCapacity = cargoCapacity;
+        return this;
+    }
+
+    public int getNumberOfAxles() {
```

Comment:
As with the getter for `cargoCapacity`, consider adding a brief comment describing the purpose of this method.